master/buildbot/changes/gitpoller.py

```diff
@@ -285,6 +285,17 @@ def process(git_output):
             return git_output
         return d
 
+    def _get_commit_committer(self, rev):
```

Comment:
Could we use @defer.inlineCallbacks for new code?

Reply:
Not sure how to do it (I'm not very familiar with twisted). Could you tell me?

Reply:
http://lmgtfy.com/?q=twisted+inlinecallbacks (2nd answer is the tutorial)

+ there are tons of examples in the buildbot code

Reply:
@tardyp @p12tic Is this the right way?
```py
@defer.inlineCallbacks
def _get_commit_committer(self, rev):
    args = ['--no-walk', r'--format=%aN <%aE>', rev, '--']
    try:
        res = yield self._dovccmd('log', args, path=self.workdir)
    except:
        raise EnvironmentError('could not get commit committer for rev')
    return res
```